_field-types/supported-field-types/knn-vector.md

```diff
@@ -163,4 +163,40 @@ GET test-index/_search
   }
 }
 ```
-{% include copy-curl.html %}
\ No newline at end of file
+{% include copy-curl.html %}
+
+### Quantization techniques
```

Comment:
Can we also include the pseudo code used for ConsineSimilarity SpaceType and the reason behind using a different technique for different type of dataset(angular) which emphasizes the importance of a quantization technique based on type of data?

Reply:
@naveentatikonda Done. Please review when you get a chance.

Reply:
@kolchfa-aws Added few comments. Pls take a look